pages/index.js

```diff
@@ -17,7 +17,9 @@ export default function Home() {
       city: "Gelting",
       zip: "24395",
       street: "Strandweg 1",
-      coordinates: null,
+      coordinates: "54.755147, 9.878069",
```

Comment:
I would code this array  in a data.js and only import it hier. But this is only my personal opinion